drltrace/CMakeLists.txt

```diff
@@ -70,7 +68,12 @@ use_DynamoRIO_extension(drltracelib drwrap_static)
 use_DynamoRIO_extension(drltracelib drx_static)
 use_DynamoRIO_extension(drltracelib drsyscall_static)
 use_DynamoRIO_extension(drltracelib drcovlib_static)
+use_DynamoRIO_extension(drltracelib droption)
 
+if (UNIX)
+  # to avoid conflicts with new declaration with "C" linkage in utils.h
```

Comment:
So droption.h includes both `<string>` and `<string.h>`?  We should remove the `<string.h>` (and replace the strcmp or whatever is in there with a std::string comparison).